backend/src/v4/config.js

```diff
@@ -279,7 +279,7 @@ config.fileUploads = {
 	modelSizeLimit: config.uploadSizeLimit,
 	resourceSizeLimit:config.resourceUploadSizeLimit,
```

Comment:
I see it is now possible to change the priority of the ticket from the list, should the same also be possible for the ticket status?
![image](https://github.com/3drepo/3drepo.io/assets/43547703/0cca8a3c-9209-4be9-aeca-710a851e0834)